### ISSUE TITLE
Memory leak

### ISSUE THREAD

**Issue by 8HoLoN** [meganz/MEGAsync#1005]
The app seems to have a memory leak at least from version 5.4.1 and 5.5.0.
The process grows of 0.6 Mo each second and consume 10% of cpu in idle.

![image](https://github.com/user-attachments/assets/1353a293-a24d-4260-8504-5900ddea17d0)

**Comment by vtmateos** [comment]
Hi @8HoLoN ,
Could you test with latest 5.6.1 version from our website and confirm if still happens? 

You can post your comments here. 

Apart from that, could you answer the following questions?:
- Do you have any sync or backup in place?
- Could you provide a rough estimate about the number of cloud items?
- Could you describe the steps you made until you get to that point? A simple login with credentials, login with existing session, setting syncs and/or backup...,

Thanks.

**Comment by 8HoLoN** [comment]
I have installed the 5.6.1 version. The problem is still here.

- sync
- 8000 elements
- just launch the app (or auto launch at the start of the pc) with a already logged account

![image](https://github.com/user-attachments/assets/12dd0bd6-6ed6-4acc-8159-0a4eac624529)

**Comment by vtmateos** [comment]
Hi,
Well, the figures are much more reduced than your original report - 19551,7 Mo vs 730,5 Mo. We will keep an eye and forward these feedback so the developer can perform some memory check ups.

**Comment by 8HoLoN** [comment]
Yes, I have waited maybe 20 min this time. The initial (normal) RAM comsuption is about 60Mo

**Comment by 8HoLoN** [comment]
Hi, @vtmateos 
I have installed the 5.9.0 version, the problem is still there. Maybe be it is due to the fact that I have recursive folder with symlink (junction) ?

**Comment by vtmateos** [comment]
Hi,
Could you send new log files to the ref GVR-408-63660 case? We will check out and start debugging from there.
Thanks for your cooperation.

**Comment by 8HoLoN** [comment]
Where should I send it ? (I got the ref GVR-408-63660 with notification email, is there a website associated with it? Which url ?)
Moreover, I checked the log file and it contains sensitives data as the credentials of the user and its private key.

**Comment by vtmateos** [comment]
Hi,
Please feel free to remove any message from the log file before sending. 
After that, you can either reply to that last email attaching new log files or create a new issue referencing GVR-408-63660. Let me know here when you complete it, so i can follow up your case.
Thanks

**Comment by 8HoLoN** [comment]
Hi @vtmateos,

I replied to the notification email (support@mega.nz) with the log file attached and referencing GVR-408-63660 in the emaill.

Regads,

**Comment by vtmateos** [comment]
Hi @8HoLoN ,
We have been analysing your issue and it could be related with the explorer integration.
It would be useful if you can disable these two options in settings, general:

- Enable file explorer integration
- Enable sync status icons

and retry your tests. Let us know whether with those options disabled the problem still persists.

Thanks.

**Comment by 8HoLoN** [comment]
Hi @vtmateos,
I have tested version 5.12.0 with following features disabled :

- file explorer integration
- sync status icons

The problem still persists.

![Image](https://github.com/user-attachments/assets/6aa56c73-ae2c-4b32-b71b-c95a210d9634)

**Comment by vtmateos** [comment]
Hi @8HoLoN ,

Could you create a log file with those options disabled and send to us through our support team? Just confirm when it is done and we well analyse it.

Thanks for your cooperation.

**Comment by vtmateos** [comment]
Hi @8HoLoN ,
Did you get a chance to create the log file and send to our support team with id GVR-408-63660?
Thanks.

**Comment by 8HoLoN** [comment]
Sorry but my windows pc run out of live few week ago, I am now on Ubuntu. So could not produce any more logs other than ones I already forwarded. 🫤
